frontend/packages/pipelines-plugin/src/components/pipelineruns/PipelineRunParametersForm.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { Form } from '@patternfly/react-core';
+import { Formik, FormikProps, FormikValues } from 'formik';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { history } from '@console/internal/components/utils';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { FormFooter } from '@console/shared';
+import PipelineRunParameters from './PipelineRunParameters';
+
+export interface PipelineRunFormProps {
+  obj: K8sResourceKind;
+}
+
+const PipelineRunParametersForm: React.FC<FormikProps<FormikValues>> = () => {
+  const { t } = useTranslation();
+  const handleCancel = () => history.goBack();
+  return (
+    <Form>
+      <div className="co-m-pane__body">
+        <PipelineRunParameters fieldName="parameters" />
+      </div>
+      <FormFooter
+        handleCancel={handleCancel}
+        cancelLabel={t('pipelines-plugin~Cancel')}
+        hideSubmit
+        sticky
+      />
+    </Form>
```

Comment:
In PipelineRunParameters you define all fields as read-only. Why do we need a form and cancel button on this tab?

Reply:
My initial thought was the same. Removed the Form and cancel button.

Reply:
@vikram-raj I'm confused, there is still a `PipelineRunParametersForm.tsx` in the change??